readthedocs/doc_builder/python_environments.py

```diff
@@ -43,7 +43,7 @@ def delete_existing_build_dir(self):
     def install_package(self):
         setup_path = os.path.join(self.checkout_path, 'setup.py')
         if os.path.isfile(setup_path) and self.config.install_project:
-            if getattr(settings, 'USE_PIP_INSTALL', False):
```

Comment:
This is actually an operator option, not a user option currently. I agree that having it be configurable by the user makes sense. :+1: